dftimewolf/lib/collectors/gcp_logging.py

```diff
@@ -85,16 +83,15 @@ def Process(self):
               self._filter_expression, exception), critical=True)
       return
 
-    except AccessTokenRefreshError as exception:
+    except google_auth_exceptions.RefreshError as exception:
       self.state.AddError(
           'Something is wrong with your gcloud access token.')
       # TODO: determine if exception should be converted into a string as
       # elsewhere in the codebase.
       self.state.AddError(exception, critical=True)
       return
 
-    except (ApplicationDefaultCredentialsError,
-            google_auth_exceptions.DefaultCredentialsError) as exception:
+    except google_auth_exceptions.DefaultCredentialsError as exception:
```

Comment:
Could we also catch `google_auth_exceptions.RefreshError` here? Both errors are probably solved by refreshing Application Default Credentials.

Reply:
Sure, done.